generator/recipe.go

```diff
@@ -15,9 +15,13 @@ var file embed.FS
 
 // Template represents the template for generating a recipe.
 type Template struct {
```

Comment:
any concern with making this struct private? technically change to this struct is also breaking change..

Reply:
no concern atm, since it is only being used inside the package, we can make it as private.